src/procedures/procedures.11tydata.yaml

```diff
@@ -367,6 +367,152 @@ proceduresInstallingHAOS:
           <li>The Home Assistant frontend guides you through the initial onboarding.</li>
           <li class="info">For further information, refer to the instructions provided under <a href="https://www.home-assistant.io/getting-started/onboarding/" target="_blank">https://www.home-assistant.io/getting-started/onboarding/</a>.</li></br>          
         </ul>
+proceduresInstallingHAOSrpiboot:
+  title: Running rpiboot
+  steps:
+    - title: Install rpiboot on your computer.
+      image: source_code.png
+      content: |
+        <ul>
+        <li>rpiboot is a command line utility for booting a Raspberry Pi in USB device mode. This is needed because the CM5 cannot be booted directly from USB.</li>
```

Comment:
```suggestion
        <li>rpiboot is a command line utility for booting a Raspberry Pi in USB device mode. This is needed because the CM5 cannot boot from the USB 2.0 ports of the Yellow.</li>
```